### PR TITLE
Tool docs

### PR DESCRIPTION
**Pull request by benreynwar**
Some very minor additions to the tools documentation.  Please double-check the definition of 'tile-offset'

### CODE REVIEW

**Comment by mithro** [comment]
@kgugala - Who would be the best to review this? I was thinking @mkurc-ant or @acomodi but wasn't sure?

File: docs/db_dev_process/parts.rst [hunk 1]
```diff
@@ -4,3 +4,58 @@ Tools
...
+    <....>
+    tag HCLK_IOI3.LVDS_25_IN_USE 0
+    tag HCLK_IOI3.ONLY_DIFF_IN_USE 0
+    <...>
+    seg 00001C80_050
+    bit 38_15
+    bit 38_26
+
+where `seg <base_frame_address> <tile_offset>` indicates how to address the tile,
+and `bit <frame_address_offset> <bit_position>` indicates the position of the bit
```
Comment (mkurc-ant) [inline docs/db_dev_process/parts.rst:45]: There is always `_` between those fields.

Comment (benreynwar) [reply docs/db_dev_process/parts.rst]: Fixed.  Sorry that took so long, I missed this going by.

Comment (mkurc-ant) [reply docs/db_dev_process/parts.rst]: Alright, thanks. Please add a signoff to the recent commit and we should be good for merging.

**Comment by mithro** [comment]
@mkurc-ant - Should this be merged?

**Comment by mkurc-ant** [comment]
Ok, merging.

**Pull request merged**